Leadfeeder/ember-cli-swiper
# [#21005] Fix dependency vulnerabilities

Closes https://github.com/Leadfeeder/issue-tracker/issues/21005:
- Upgrades dependencies
- Removes unused dependencies

### How to test
See "how to test" instructions in [this PR](https://github.com/Leadfeeder/francium/pull/4645)
